app/helpers/application_helper.rb

```diff
@@ -364,4 +364,14 @@ def has_out_of_date_configuration?(service)
     return unless service
     service.pending_affecting_changes?
   end
+
+  #TODO: Tests pending for this
+  def rails_asset_host_url
+    asset_host_enabled = Rails.configuration.asset_host.present?
```

Comment:
Is `asset_host_enabled` needed at all?

Reply:
I think so, `asset_host_enabled` contains the lambda function at [/config/environments/production.rb](https://github.com/3scale/porta/blob/0f7e55f730c38fca30ff005c0b32be956b1bf54f/config/environments/production.rb#L112). That can be `:nil` when `config.assets.digest != true`. It could happen that `settings.yml:asset_host` is not null but the CDN is disabled, in that case we don't want Webpack to prepend the url